lighthouse-core/gather/gather-runner.js

```diff
@@ -374,7 +374,9 @@ class GatherRunner {
     return {
       fetchTime: (new Date()).toJSON(),
       LighthouseRunWarnings: [],
-      UserAgent: await options.driver.getUserAgent(),
+      HostUserAgent: await options.driver.getUserAgent(),
+      NetworkUserAgent: '', // updated later
+      BenchmarkIndex: await options.driver.getBenchmarkIndex(),
```

Comment:
do we want to run the benchmark in the target page (vs `about:blank` or whatever we end up on)? Seems like what's going on in the page could affect the results

Reply:
yeah I think we'll want to experiment with when this gets evaluated, for example at the moment I think it's being affected by being run while Chrome is still being setup. The values I'm seeing are noticably lower than when I run it after page load.

Reply:
good call moving it just after load blank which does the 300 ms waiting already much improves the estimate 👍

Reply:
> good call moving it just after load blank which does the 300 ms waiting already much improves the estimate 👍

ha, I was going to come back later and say maybe it doesn't make a difference and isn't worth it, so good to hear :)

I guess it's possible it'll cause more issues with moving off using `about:blank` in the future, but that's a problem for another day :)